DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces_overview/csr-pipes/src/vector_add.cpp

```diff
@@ -0,0 +1,123 @@
+#include <iostream>
+
+// oneAPI headers
+#include <sycl/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
```

Comment:
```suggestion

#include "exception_handler.hpp"

```

Reply:
From Yohann's comment on my sample, the "exception_handler" header is needed to support nice exception handling on windows, so I think we'll probably need to add it here